DESCRIPTION

```diff
@@ -1,5 +1,5 @@
 Package: epidist
-Title: Estimate epidemiological delay distributions for infectious diseases 
+Title: Estimate epidemiological delay distributions with brms 
```

Comment:
Just a drive-by review


Title should be capitalised. See [here](https://r-pkgs.org/description.html).